advanced/is-with-analytics/scripts/azure-deploy.sh

```diff
@@ -0,0 +1,191 @@
+
+# create a new Kubernetes Namespace
+${KUBERNETES_CLIENT} create namespace wso2
+
+# create a new service account in 'wso2' Kubernetes Namespace
+${KUBERNETES_CLIENT} create serviceaccount wso2svc-account -n wso2
+
+# switch the context to new 'wso2' namespace
+${KUBERNETES_CLIENT} config set-context $(${KUBERNETES_CLIENT} config current-context) --namespace=wso2
+
+if [[ ${HAS_SUBSCRIPTION} -eq 0 ]]; then
+    # create a Kubernetes Secret for passing WSO2 Private Docker Registry credentials
+    ${KUBERNETES_CLIENT} create secret docker-registry wso2creds --docker-server=docker.wso2.com --docker-username=${WSO2_SUBSCRIPTION_USERNAME} --docker-password=${WSO2_SUBSCRIPTION_PASSWORD} --docker-email=${WSO2_SUBSCRIPTION_USERNAME}
+fi
+# create Kubernetes Role and Role Binding necessary for the Kubernetes API requests made from Kubernetes membership scheme
+${KUBERNETES_CLIENT} create -f ../../rbac/rbac.yaml
+
+# create Kubernetes ConfigMaps
+echoBold 'Creating ConfigMaps...'
+${KUBERNETES_CLIENT} create configmap identity-server-conf --from-file=../confs/is/conf/
+${KUBERNETES_CLIENT} create configmap identity-server-conf-axis2 --from-file=../confs/is/conf/axis2/
+${KUBERNETES_CLIENT} create configmap identity-server-conf-datasources --from-file=../confs/is/conf/datasources/
+${KUBERNETES_CLIENT} create configmap identity-server-conf-identity --from-file=../confs/is/conf/identity/
+${KUBERNETES_CLIENT} create configmap identity-server-conf-event-publishers --from-file=../confs/is/deployment/server/eventpublishers/
+${KUBERNETES_CLIENT} create configmap identity-server-conf-tomcat --from-file=../confs/is/conf/tomcat/
+${KUBERNETES_CLIENT} create configmap is-analytics-worker-conf --from-file=../confs/is-analytics-worker/conf/worker
+${KUBERNETES_CLIENT} create configmap is-analytics-dashboard-conf --from-file=../confs/is-analytics-dashboard/conf/dashboard
+${KUBERNETES_CLIENT} create configmap mysql-dbscripts --from-file=../extras/confs/rdbms/mysql/dbscripts/
+
+echoBold 'Deploying the Kubernetes Services...'
+${KUBERNETES_CLIENT} create -f ../extras/rdbms/mysql/mysql-service.yaml
+${KUBERNETES_CLIENT} create -f ../is/identity-server-service.yaml
+${KUBERNETES_CLIENT} create -f ../is-analytics-worker/identity-server-analytics-worker-service.yaml
+${KUBERNETES_CLIENT} create -f ../is-analytics-dashboard/identity-server-analytics-dashboard-service.yaml
+sleep 10s
+
+echoBold 'Creating Azure storage classes'
+${KUBERNETES_CLIENT} apply -f ../../../azure/rbac.yaml
+${KUBERNETES_CLIENT} apply -f ../../../azure/storage-class.yaml
+${KUBERNETES_CLIENT} apply -f ../../../azure/mysql-storage-class.yaml
+
+# persistent storage
+echoBold 'Creating persistent volume and volume claim...'
+${KUBERNETES_CLIENT} create -f ../is/identity-server-volume-claims-azure.yaml
+${KUBERNETES_CLIENT} create -f ../extras/rdbms/mysql/mysql-persistent-volume-claim-azure.yaml
+# ${KUBERNETES_CLIENT} create -f ../volumes/persistent-volumes.yaml
```

Comment:
@Aaquiff as per my understanding, these commented out, deployment commands for Kubernetes Persistent Storage resource definitions may not be required within the Azure specific deployment management script `azure-deploy.sh`.

If a user intends to use NFS, he/she can use the platform independent `deploy.sh` script.

**Note**: Same applies to `undeploy.sh` script.

Hence, as per my understanding it is safe to remove these commented out deployment commands for unused definition files rather than leaving them commented out. WDYT?

Reply:
@chirangaalwis Removed the commented out volume deployments from the scripts